### ISSUE TITLE
Error when using Python

### ISSUE THREAD

**Issue by IsalineMs** [jdrudolph/photon#32]
Hi Jan, 
I keep getting an error message after launching photon. I believe it is because there is a white space in the output path? Would you know how to fix this? Thanks !
![121814522_3354232647993088_343279793870492232_n](https://user-images.githubusercontent.com/73093586/96404153-e4ffee80-1225-11eb-9c2f-d1910c3a9a2b.jpg)

**Comment by jdrudolph** [comment]
might be a problem with spaces in the user folder name. is that the case?

**Comment by IsalineMs** [comment]
Yes there is one. Is it possible to fix this without changing the user name? Thanks
